### ISSUE TITLE
PredicateExpressions for String is only available for FOUNDATION_FRAMEWORK

### ISSUE THREAD

**Issue by axelandersson** [apple/swift-foundation#540]
Hi,

The predicate expressions `StringLocalizedStandardContains` and `StringLocalizedCompare` are under `#if FOUNDATION_FRAMEWORK`.

I would like to use these on Linux as well.

**Comment by jmschonfeld** [comment]
Those two are currently unavailable because the APIs that they represent (`String.localizedStandardContains` and `String.localizedStandardCompare`) are not yet available in the package. We can add the `Predicate` operators as soon as the actual APIs are available. It looks like we have https://github.com/apple/swift-foundation/issues/284 open to track localized comparison, @itingliu do we have another radar tracking bringing over the string contains functions (localized in particular)?

**Comment by itingliu** [comment]
Yes we have a radar for implementing localized contain: 104656368. We can create an issue as well if we think that helps keeping things on track.

**Comment by jhansbo** [comment]
Are there any plans to address this issue on Linux in a not too distant future ?
